src/main/scala/DocsPlugin.scala

```diff
@@ -0,0 +1,117 @@
+package verizon.build
+
+import sbt._, Keys._
+import scala.language.existentials
+
+object DocsPlugin extends AutoPlugin { self =>
+  object autoImport {
+    val preStageSiteDirectory = SettingKey[File]("pre-stage-site-directory")
+    val siteStageDirectory    = SettingKey[File]("site-stage-directory")
+    val copySiteToStage       = TaskKey[Unit]("copy-site-to-stage")
+    val gitRemoteHost         = SettingKey[String]("git-remote-host", "the host name of the remote git server (ex: github.com)")
+    val githubOrg             = SettingKey[String]("github-org", "the organization name of a project within GitHub (ex: Verizon)")
+    val githubRepoName        = SettingKey[String]("github-repo-name", "the repository name of a project within GitHub (ex: knobs)")
+    val siteUnidocPath        = SettingKey[String]("site-unidoc-path", "the path of the site at which the scaladoc should be located (ex: api)")
+  }
+
+  import autoImport._
+
+  override def trigger = noTrigger
+
+  override def requires =
+    RigPlugin &&
+    DisablePublishingPlugin &&
+    com.typesafe.sbt.site.SitePlugin &&
+    com.typesafe.sbt.sbtghpages.GhpagesPlugin &&
+    com.typesafe.sbt.site.hugo.HugoPlugin &&
+    sbtunidoc.ScalaUnidocPlugin &&
+    tut.TutPlugin
+
+  override lazy val projectSettings = self.settings
+
+  import java.net.URI
+  // site
+  import com.typesafe.sbt.site.SitePreviewPlugin.autoImport.previewFixedPort
+  import com.typesafe.sbt.site.SitePlugin, SitePlugin.autoImport._
+  import com.typesafe.sbt.site.hugo.HugoPlugin, HugoPlugin.autoImport._
+  import com.typesafe.sbt.sbtghpages.GhpagesPlugin, GhpagesPlugin.autoImport._
+  import com.typesafe.sbt.SbtGit.git
+  // unidoc
+  import sbtunidoc.BaseUnidocPlugin, BaseUnidocPlugin.autoImport._
+  import sbtunidoc.ScalaUnidocPlugin, ScalaUnidocPlugin.autoImport._
+  // release
+  import sbtrelease.ReleasePlugin.autoImport._
+  // tut
+  import tut.TutPlugin.autoImport._
+  // rig
+  import RigPlugin.autoImport._
+  import scoverage.ScoverageKeys.coverageEnabled
+
+  /**
+   * These settings should only ever be used in isolation, on a documentation
+   * specific module of your project.
+   */
+  def settings =
+    Seq(
+      coverageEnabled := false,
+      gitRemoteHost := "github.com",
```

Comment:
Yes.  That key is defined by this plugin, so this is where we're making the the default.